json/json_test.go

```diff
@@ -336,7 +336,7 @@ func TestCodec(t *testing.T) {
 		t.Run(testName(v1), func(t *testing.T) {
 			v2 := newValue(v1)
 
-			a, err := json.MarshalIndent(v1, "", "\t")
+			a, err := MarshalIndent(v1, "", "\t")
```

Comment:
I think this was intended to use the stdlib json package. Below it compares `a` (from the stdlib json) to `b` (from NewEncoder() from this package)

Reply:
Possibly, this test hasn't been touched in a while, I think it predates adding the `MarshalIndent` function to this package. We still rely on `json.Ident` form stdlib so the behavior didn't change.

I'll revert if you think it's serves a better purpose the way it was 👍

Reply:
I see, that also makes sense: comparing `MashalIndent` to `Encoder.SetIndent`. I'm no expert here, just trying to help :)